QueAns/tests.py

```diff
@@ -0,0 +1,16 @@
+#couldn't test it
+from django.test import Client
+
+c = Client()
+response = c.post('/upvote/',{'user_id':'prakhar_96','question_id':'1'})
```

Comment:
These are the user's that are created in your system right, now what if I run this same test on my system, this will fail(assert below will fail), right? Because I don't have these users in my system.